### PR TITLE
chore: change broken YARD-links to github friendly md links in handwritten libraries 

### PR DESCRIPTION
**Pull request by trambui09**
Follow up PR to change broken YARD-links in the markdown files to github-friendly md links

### CODE REVIEW

File: google-cloud-compute-v1/README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ In order to use this library, you first need to go through the following steps:
 
 1. [Select or create a Cloud Platform project.](https://console.cloud.google.com/project)
 1. [Enable billing for your project.](https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project)
-1. {file:AUTHENTICATION.md Set up authentication.}
+2. [Set up authentication.](AUTHENTICATION.md)
```
Comment (dazuma) [inline google-cloud-compute-v1/README.md:23]: Leave the step number at 1. (Markdown will renumber them when it renders the page.)

File: google-cloud-asset-v1beta1/README.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ In order to use this library, you first need to go through the following steps:
 1. [Select or create a Cloud Platform project.](https://console.cloud.google.com/project)
 1. [Enable billing for your project.](https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project)
 1. [Enable the API.](https://console.cloud.google.com/apis/library/cloudasset.googleapis.com)
-1. {file:AUTHENTICATION.md Set up authentication.}
```
Comment (dazuma) [inline google-cloud-asset-v1beta1/README.md:22]: This library is a special case. It's discontinued (and we probably should delete it, but I still need to figure out how to do so safely), so leave this file as is.

File: google-cloud-firestore/OVERVIEW.md [hunk 3]
```diff
@@ -490,4 +490,4 @@ the entire collection or sub-collection.
 ## Additional information
 
 Google Firestore can be configured to use gRPC's logging. To learn more, see the
-{file:LOGGING.md Logging guide}.
+[Logging guide](file:LOGGING.md).
```
Comment (dazuma) [inline google-cloud-firestore/OVERVIEW.md:493]: Whoops, left the "file:" there.

And actually it looks like there are a bunch of cases of this. (Maybe all for LOGGING.md?) Please fix all of them.

File: google-cloud-spanner/OVERVIEW.md [hunk 4]
```diff
@@ -316,9 +316,9 @@ spanner = Google::Cloud::Spanner.new
 instance = spanner.instance "my-instance"
 
 instance.delete
-````
```
Comment (dazuma) [inline google-cloud-spanner/OVERVIEW.md:319]: Thanks for finding this!

**Review by dazuma — CHANGES_REQUESTED** [review]

File: google-cloud-firestore/OVERVIEW.md (hunk 3 above)
Comment (trambui09) [reply google-cloud-firestore/OVERVIEW.md:493]: Good catch! And this is why we have code reviews haha

**Comment by trambui09** [comment]
Made the proper edits for the LOGGING.md links!

**Review by dazuma — APPROVED** [review]
Thanks Tram!

**Comment by dazuma** [comment]
We'll go ahead and merge this. Existing test failures are unrelated stuff that should be fixed separately.

**Pull request merged**